src/Array.mo

```diff
@@ -23,6 +23,7 @@ module {
     return true;
   };
   /// Append the values of two input arrays
+  /// @deprecated Array.append will be removed soon; use a Buffer, and Buffer.append, instead.
```

Comment:
Would be good to say why it's deprecated.

Reply:
Okay.  Changed "will be removed" to "has critical performance flaws".

BTW -- In the world of Rust, they often link to PRs and issues for people to read about on Github.  We don't do that, except for error codes.  We could do that here, though, and in similar situations.